README.md

```diff
@@ -75,7 +75,18 @@ An example of a valid `x-trapi` extension in YAML is:
       version: "1.3.0"
       operations:
       - lookup
-      test_data_location: "https://github.com/biothings/biothings.api/blob/master/test_data"  # just a fictional example link
+      test_data_location: { ## complex test_data_location example, but could still be one single URL string as well
+        "default": { ## the "default" test data set applied against all other defined "x-maturity" environments in the entry's "servers" block Registry, other than "development", which is explicitly specified here below.
+          "url": [
+            "https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/sri-test-service-provider.json",
+            "https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/sri-test-multiomics.json",
+            "https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/sri-test-text-mining.json"
+          ]
+        },
+        "development": {
+          "url": "https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/biolink3/biothings_explorer/qualifier-sri-test-service.json"
+        }
+      }
```

Comment:
This example needs to be revised to yaml format. 

Also, I'm not sure that the example fully reflects the schema (like the required tag). Perhaps the example below may work better?

```
tags:
- name: trapi        ## required tag
info:
  x-trapi:
    version: 1.3.0     ## required property
    operations:        ## required property
      - lookup
    test_data_location:    ## example of the object format
      development:
        url: "https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/biolink3/biothings_explorer/qualifier-sri-test-service.json"
      default:
        url:
        - "https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/sri-test-service-provider.json"
        - "https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/sri-test-multiomics.json"
        - "https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/sri-test-text-mining.json"
```

Reply:
Has been addressed in later commit